seo-learning/seo-quiz.md

```diff
@@ -0,0 +1,205 @@
+Search Engine Optimization(SEO)
+-------------------------------
+
+1. What term is typically used to explain the shuffling of positions in search engine results in between significant updates?
+A. Flux.
+B. Waves.
+C. Swaying.
+D. Shuffling.
+
+
+Answer:- (A)
```

Comment:
Would be great to keep following common style guide for answers using checkbox markdown format. You can look trow how it was implemented in other `quiz.md` files.